src/runtime/runtime-simd.cc

```diff
@@ -329,7 +329,6 @@ SIMD128_UNARY_FUNCTIONS(DECLARE_SIMD_UNARY_FUNCTION)
 
 template<typename T1, typename T2>
 inline void BitsTo(T1 s, T2* t) {
```

Comment:
just for compilation, if remove BitsTo, compilation will be failed because  the program doesn't check condition judgement in the expansion of macro.